### DISCUSSION TITLE
Automation to create app from custom template

### DISCUSSION THREAD

**Discussion by snvgglebear** [Budibase/budibase#6010] [Help]
I want to create an automation that creates a new app as a copy of an existing app, but updates some of the database connection information. (Such as the database/schema to connect to). How would I do this?

**Comment by melohagan** [comment]
Hi @gondilon 

I don't think this is possible at the moment. The public API allows you to get the meta data of an app, but not the export file. 
Again there is no way to update database connection info via the public api. 

It might be possible to use an automation JS / Bash block to call the internal endpoints to get the app export, update the content with the connection info you want, and create a new app from there, but I'm not sure how to accomplish this.

It may be easier to extend the available public endpoints in the codebase.
